tasks/sep/006-1-sc-changes/SignFromJson.s.sol

```diff
@@ -437,21 +438,37 @@ contract SignFromJson is OriginalSignFromJson {
         require(securityCouncilSafe.getThreshold() == 3, "checkSecurityCouncilSafe-301");
     }
 
+    function checkOwnershipConfiguration() internal {
+        (address[] memory modules,) = ModuleManager(guardianSafe).getModulesPaginated(SENTINEL_MODULE, 3);
+        deputyGuardianModule = IDeputyGuardianModuleFetcher(modules[0]);
+        require(modules.length == 1, "checkOwnershipConfiguration-50");
+        require(address(deputyGuardianModule) == expectedDeputyGuardianModule, "checkOwnershipConfiguration-100");
```

Comment:
not technically incorrect, but I'd change a few things in the modules check here (similarly in the next one). 

1. call `getModulesPaginated(SENTINEL_MODULE, 1)` (each only has one module).
2. get the `next` value and verify that it matches the `SENTINEL_MODULE`

Reply:
Done in e444512 which also changes to the new contract addresses